native/src/main/scala/slinky/native/Keyboard.scala

```diff
@@ -0,0 +1,16 @@
+package slinky.native
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSImport
+
+@js.native
+@JSImport("react-native", "Keyboard")
+object Keyboard extends js.Object {
+  def addListener(eventName: String, callBack: () => Unit): Unit = js.native
```

Comment:
This (and the equivalent for `removeListener`) should take a `js.Function0[Unit]` since this is a low-level facade. `() => Unit` refers to the Scala representation of functions, which can sometimes be incompatible with the JS representation in odd ways.

Reply:
Thanks for the review and tip, @shadaj! Please see revisions.